src/test/java/com/wikia/webdriver/elements/communities/mobile/components/PortableInfoboxObject.java

```diff
@@ -40,7 +40,7 @@
   @FindBy(css = ".pi-image-collection")
   private WebElement imageInCollection;
 
-  @FindBy(css = ".portable-infobox .article-media-linked-gallery img")
+  @FindBy(css = ".article-media-gallery__column")
```

Comment:
this selector matches the list of columns in the gallery while variable name states imageList, which is true? also this will match columns from all galeries in article